### DISCUSSION TITLE
Sideloading not working

### DISCUSSION THREAD

**Discussion by phxfoto** [requarks/wiki#5389] [Error / Bug Report]
I have configured our Linux Ubuntu 20 system to both run on line and off line. When online I keep getting a "an error has occurred" message with "request to https;//graph.requarks.io files, ...." I can ping graph.requarks.io so I don't know why this proving an  error. Additionally. If I set the system to run local and download the locale.json and en.json files I continue to get the same errors.
 
![Screen Shot 2022-06-10 at 3 37 45 PM](https://user-images.githubusercontent.com/107282586/173159275-5226db08-03d3-48ee-9885-374133d7c262.jpg)

![Screen Shot 2022-06-10 at 3 38 01 PM](https://user-images.githubusercontent.com/107282586/173159284-04e0d61d-c779-48a1-88a9-8490ce8fc26d.jpg)

I am not sure what is going on. It's a VM running the latest node.js (16) and latest postgresSQL. I can add content without errors (or so it seems).

**Comment by NGPixel** [comment]
You need to set `offline: true` in your config.yml as well

**Reply by azfotoman** [reply]
I have offline set to true and restarted the server with no change. I've also tried to not use side loading as the system is not firewalled from the web, but it still won't download languages.
